tmail-backend/blob/blobid-list/src/main/scala/com/linagora/tmail/blob/blobid/list/postgres/PostgresBlobIdListModule.scala

```diff
@@ -0,0 +1,23 @@
+package com.linagora.tmail.blob.blobid.list.postgres
+
+import org.apache.james.backends.postgres.{PostgresModule, PostgresTable}
+import org.jooq.impl.{DSL, SQLDataType}
+import org.jooq.{Field, Record, Table}
+
+object PostgresBlobIdListModule {
+  val TABLE_NAME: Table[Record] = DSL.table("blob_id_list")
```

Comment:
I has commented on the issue ticket:
https://github.com/linagora/tmail-backend/issues/922#issuecomment-2021791817